components/Modal/PendingChanges.js

```diff
@@ -211,44 +210,74 @@ class PendingChanges extends React.Component {
                       </Button>
                     </OverlayTrigger>
                     <ul className="list-group">
-                      {this.props.blueprint.workspacePendingChanges.addedChanges.length !== 0 && (
-                        <li className="list-group-item">
-                          <div className="row">
-                            <div className="col-sm-3">Added</div>
-                            <div className="col-sm-9">
-                              <ul className="list-unstyled">
-                                {this.props.blueprint.workspacePendingChanges.addedChanges.map(componentUpdated => (
-                                  <li key={componentUpdated.new.Package.name}>
-                                    <strong>
-                                      {Object.values(componentUpdated.new)[0].name}-
-                                      {Object.values(componentUpdated.new)[0].version}
-                                    </strong>
-                                  </li>
-                                ))}
-                              </ul>
+                      {this.props.blueprint.workspacePendingChanges.map(change => (
+                        <li
+                          className="list-group-item"
+                          key={!change.componentNew ? change.componentOld : change.componentNew}
```

Comment:
Object inside `workspacePendingChanges` array has "name" and "version" keys. 
But object inside `localPendingChanges`. array does not. It would be better to have same object structure.
![componentnew](https://user-images.githubusercontent.com/672230/51234546-fb6d7c00-19a7-11e9-9190-123f215fc23f.png)

Should be changed to
```
key={
   !change.componentNew
   ? `${change.componentOld.name}-${change.componentOld.version}`
   : `${change.componentNew.name}-${change.componentNew.version}`
}
```

Reply:
Yes, this is definitely a change I'd like to make. I'd also like to separate the name and version as you mention below. I was thinking these changes could be a separate PR, since this is already pretty big.